doc/sphinx/practical-tools/utilities.rst

```diff
@@ -973,13 +973,13 @@ line timing data:
     .. note::
       This target requires python to build the table.
 
-+ ``PROFILE=1``
++ ``PROFILING=1``
   passing this variable or setting it in the environment will cause
-  ``make`` to use ``rocq c -profile`` to write to a ``.v.prof.json``
+  ``make`` to use ``rocq c -profile`` to write to a ``.vo.prof.json``
   file for each ``.v`` file compiled, which contains :ref:`profiling`
   information.
 
-  The ``.v.prof.json`` is then compressed by ``gzip`` to a ``.v.prof.json.gz``.
+  The ``.vo.prof.json`` is then compressed by ``gzip`` to a ``.vo.prof.json.gz``.
```

Comment:
I may miss something but this seems to me like contradicting the changelog entry for https://github.com/coq/coq/pull/19428/files#diff-9600b4b30253f093df51f48179aa7fc61f3d4365cd4946d4e7559a090c91269b (but it would be in agreement with the code, so maybe the changelog entry would be the incorrect one? (but this would disagree with dev/bench/banch.sh ? I'm a bit lost))

Reply:
How does it contradict the changelog?

Reply:
I guess " **Changed:**  `coq_makefile` generates profiling info for `coqc` in `foo.vo.prof.json.gz` instead of `foo.v.prof.json.gz`" can be read as "coq_makefile now generates foo.vo.prof..." (correct) or "coq_makefile used to generate foo.vo.prof... and now generates foo.v.prof..." (incorrect)